packages/core/src/writers/schemas.ts

```diff
@@ -57,13 +58,21 @@ export const writeSchema = async ({
   path: string;
   schema: GeneratorSchema;
   target: string;
+  namingConvention: string;
   fileExtension: string;
   specKey: string;
   isRootKey: boolean;
   specsName: Record<string, string>;
   header: string;
 }) => {
-  const name = camel(schema.name);
+  let name = camel(schema.name);
```

Comment:
Please refer to this and add the const definition.
When referencing, refer to the exported const and compare instead of directly comparing strings 👍 

https://github.com/orval-labs/orval/blob/master/packages/core/src/types.ts#L185-L191